test/example_data.dart

```diff
@@ -302,22 +302,22 @@ Map<String, dynamic> _messagePropertiesFromContent(String? content, String? cont
 int _nextMessageId() => (_lastMessageId += 1 + Random().nextInt(100));
 int _lastMessageId = 1000;
 
-const defaultStreamMessageStreamId = 123;
+const defaultChannelMessageStreamId = 123;
 
-/// Construct an example stream message.
+/// Construct an example stream channel.
 ///
 /// If the message ID `id` is not given, it will be generated from a random
 /// but increasing sequence, which is shared with [dmMessage].
 /// Use an explicit `id` only if the ID needs to correspond to some other data
 /// in the test, or if the IDs need to increase in a different order from the
-/// calls to [streamMessage] and [dmMessage].
+/// calls to [channelMessage] and [dmMessage].
 ///
 /// The message will be in `stream` if given.  Otherwise,
-/// an example stream with ID `defaultStreamMessageStreamId` will be used.
+/// an example stream with ID `defaultChannelMessageStreamId` will be used.
 ///
 /// See also:
 ///  * [dmMessage], to construct an example direct message.
-StreamMessage streamMessage({
+ChannelMessage channelMessage({
```

Comment:
Because this function `eg.streamMessage` has a huge number of references (333 of them, says my IDE), its rename should go in its own commit, separate from editing other things that refer to `StreamMessage` or "stream messages".

That'd help make the changes easier to review — including self-review to catch things like "example stream channel" noted above.